### PR TITLE
Implemented edit button

### PR DESCRIPTION
**Pull request by Mav-Ivan**
https://github.com/ita-social-projects/SpaceToStudy-Client/assets/110425368/2275850a-361a-4008-8f9d-af4cc4235e8a

### CODE REVIEW

**Review by docia — APPROVED** [review]

File: package-lock.json
Comment (Olenka-Hryk) [inline package-lock.json:1]: Could you please revert these changes in the `package-lock.json` file?

File: tests/unit/containers/course-sections-list/CourseSectionsList.spec.jsx
Comment (Olenka-Hryk) [inline tests/unit/containers/course-sections-list/CourseSectionsList.spec.jsx:1]: Why must we skip this test?

**Review by Olenka-Hryk — APPROVED** [review]

File: tests/unit/containers/course-sections-list/CourseSectionsList.spec.jsx
Comment (Mav-Ivan) [reply tests/unit/containers/course-sections-list/CourseSectionsList.spec.jsx:1]: it's loading for too long

**Review by YaroslavChuiko — APPROVED** [review]

File: src/containers/my-cooperations/cooperation-activities-list/CooperationActivitiesList.tsx [hunk 1]
```diff
@@ -21,6 +23,47 @@ const CooperationActivitiesList = () => {
...
+          ]
+        } else {
+          newSections = [...sections, ...newSectionData]
+        }
+        setSectionsData(newSections)
+      }
+      addNewSectionsCourse(currentSectionIndex)
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [isAddedClicked])
```
Comment (abalanovsky) [inline src/containers/my-cooperations/cooperation-activities-list/CooperationActivitiesList.tsx:62]: gonna be fixed in #2064

**Review by abalanovsky — APPROVED** [review]
